### PR TITLE
tests: use testify to assert the tests

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Use testify lib to assert the tests

#### Which issue(s) this PR fixes:
None

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

/assign @justaugustus @saschagrunert

### CODE REVIEW

File: buoy/pkg/git/git_test.go [hunk 1]
```diff
@@ -28,30 +30,19 @@ func TestGetRepo_BasicOne(t *testing.T) {
...
-	if r.DefaultBranch != "master" {
-		t.Errorf("expected default branch to be master, got %q", r.DefaultBranch)
-	}
-	if want := 2; len(r.Branches) != want {
-		t.Errorf("expected branch count to be %d, got %d", want, len(r.Branches))
-	}
-	if want := 1; len(r.Tags) != want {
-		t.Errorf("expected tag count to be %d, got %d", want, len(r.Branches))
-	}
+	require.Equal(t, r.DefaultBranch, "master")
```
Comment (justaugustus) [inline buoy/pkg/git/git_test.go:38]: Parameterize `master` in the package for when we eventually change the default branch names.

**Review by justaugustus — CHANGES_REQUESTED** [review]
@cpanato -- One nit! Thanks for the updates :)

File: buoy/pkg/git/git_test.go (hunk 1 above)
Comment (cpanato) [reply buoy/pkg/git/git_test.go:38]: @justaugustus the DefaultBranch is filled on the fly, it is not a hardcoded variable 
see here https://github.com/kubernetes-sigs/zeitgeist/blob/master/buoy/pkg/git/git.go#L58-L60

Do you want to make a const for the test? not sure if I follow you

**Comment by cpanato** [comment]
gently ping @justaugustus to close the loop

**Comment by cpanato** [comment]
@justaugustus ready

**Pull request merged**